controls/models.py

```diff
@@ -620,6 +622,47 @@ def get_baseline_controls(self, baselines_key, baseline_name):
     def body(self):
         return self.legacy_imp_smt
 
+class OrgParams(object):
+    """
+    Represent list of organizational defined parameters. Temporary
+    class to work with default org params.
+    """
+    
+    _singleton = None
+    
+    def __new__(cls):
+        if cls._singleton is None:
+            cls._singleton = super(OrgParams, cls).__new__(cls)
+            cls._singleton.init()
+            
+        return cls._singleton
+    
+    def init(self):
+        global ORGPARAM_PATH
+        self.cache = {}
+
+        path = Path(ORGPARAM_PATH)
+        for f in path.glob("*.json"):
+            name, values = self.load_param_file(f)
+            if name in self.cache:
+                raise Exception("Duplicate default organizational parameters name {} from {}".format(name, f))
+            self.cache[name] = values
+    
+    def load_param_file(self, path):
+        with path.open("r") as json_file:
+            data = json.load(json_file)
+            if 'name' in data and 'values' in data:
+                return (data["name"], data["values"])
+            else:
+                raise Exception("Invalid organizational parameters file {}".format(path))
```

Comment:
Perhaps there is a more specific exception that can be raised other than the generic `Exception`?

Reply:
What would you suggest, @bama4 ?  Is there a GovReady-Q exception hierarchy I should be using?

Reply:
Going to carry the generic Exception categorization as technical debut @bama4, @woodt . Do you have an example exception hierarchy we could consider?